docs/screenshot-and-gif-software.md

```diff
@@ -0,0 +1,37 @@
+# Screenshot and GIF software
+
+The following list documents different screenshoting & GIF-making techniques and
+free software. We encourage you to make use of these when making front-end pull
+requests, as other contributors can see the changes you have made without having
+to checkout your branch.
+
+## Screenshot tools by platform
+### Browser 
+- [LightShot Screenshot](https://app.prntscr.com/en/index.html) (Chrome, Firefox, IE & Opera)
+
+### MacOS
+- Command-Shift-3 to capture all of the screen
+- Command-Shift-4 and drag cursor to select a specific area
+- [LightShot Screenshot](https://app.prntscr.com/en/index.html)
+
+### Windows
+- Snipping Tool (inbuilt)
+- [LightShot Screenshot](https://app.prntscr.com/en/index.html)
+
+### Linux
+- Ctrl-Shift-PrtScn for “select an area to grab” feature of gnome-screenshot.
```

Comment:
Don't know if this is case with my laptop only or others suffer with this problem too but Ctrl-Shift-PrtScn doesn't work on my laptop. May we should recommend the shortcut as a tip, like:
```
gnome-screenshot (inbuilt, you can use Ctrl-shift-PrtScn as a shortcut for it's “select an area to grab” feature.
```